### PR TITLE
Avatar Test

### PR DESCRIPTION
**Pull request by rheckart**
Styles aren't applied on the page for some reason. Don't merge this!

### CODE REVIEW

File: src/components/Avatar/index.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      setMainElement(
+        <img src={url} alt={name} onError={() => setLoadError(true)} />
+      );
+    } else {
+      setMainElement(<span>{formatInitials(name)}</span>);
+    }
+  }, [loading, loadError, name, url]);
+
+  return (
+    <div className={classNames}>
```
[the remark this answers is not in the document]
Comment (rheckart) [reply src/components/Avatar/index.js]: No I was experimenting with applying multiple class names.

File: src/components/Avatar/index.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      {mainElement}
+      <div className="badger">{badge}</div>
```
[the remark this answers is not in the document]
Comment (rheckart) [reply src/components/Avatar/index.js]: Ya this is _really_ rough test code that will be replaced. I just got stuck when my stylesheets didn't appear to be included.

**Comment by rheckart** [comment]
> I think I found part of the problem. You need to use the `classnames` package to combine the classes

Oh I guess I was/am confused. I thought the conversation yesterday was that this was all going to be ripped out and replaced. But maybe not this (classnames). :confused:

File: src/components/Avatar/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (mdespuits) [reply src/components/Avatar/index.js]: Right. Passing an array will not add these classes properly. We use the [`classnames` ](https://jedwatson.github.io/classnames/) package to build out lists of classes to apply.

**Pull request closed without merging**